schemas/representations.md

```diff
@@ -51,27 +69,25 @@ Link is also not described in this section, as it's a rather unique business.)
 	- `listpairs` representation -- transcribes to `list` (of lists) in the Data Model.
 - Enum
 	- `string` representation -- the default -- transcribes the enum values as `string` in the Data Model.
+	- `int` representation -- transcribes the enum values as `int` in the Data Model.
 
 Each of these representation strategies will be explored in detail in the
 "Representation Strategy Reference" section, below.
 
-
-Representation Strategy Reference
----------------------------------
+## Representation Strategy Reference
 
 In this section, we'll go into detail on each representation strategy,
 and include examples of both schemas using it and data matching it.
 See the "Kinds and their Representations" section for an introduction to
 representation strategies.
 
-
 ### struct map representation
 
 Map representation of structs means that the struct is represented as a map,
 where the keys are the names of the struct fields.  This is a common and
 natural way to represent structs; it's thus their default representation.
 
-#### struct map representation example
```

Comment:
undesirable.

I know these read redundantly as heck, and I'm not a huge fan of that either... but they are as such so that `thepage.md#struct-map-representation-example` works.  :/

Reply:
we could try to do our own `<a ...>` tags, but the little get-me-a-link button that github draws wouldn't work with that either, sadly.

Reply:
yes, but they suck verbose like that, so nice hierarchy is more desirable to me

Reply:
how about just removing that level of heading and going with bolded `**Example**`, that renders nicely in MD

Reply:
See latest fixup commit and view that: https://github.com/ipld/specs/blob/19691d41261bfda89a4eab63267e057aa21b9ddd/schemas/representations.md

The sections are small enough that linking straight into an example isn't so helpful. Just link to the section.